client/scss/components/_footer_nav.scss

```diff
@@ -1,3 +1,5 @@
+//* @define footer-nav
```

Comment:
Why do we need this if it's defined below?
I am still battling to see what this is for?